query_execution/QueryContext.hpp

```diff
@@ -195,6 +195,17 @@ class QueryContext {
   }
 
   /**
+   * @brief Destroy the given aggregation state.
+   *
+   * @param id The ID of the AggregationOperationState to destroy.
+   **/
+  inline void destroyAggregationState(const aggregation_state_id id) {
```

Comment:
As a side node for the new `DestroyAggregationStateWorkOrder`, we don't need this new method. Instead, using `releaseAggregationState` and the `unique_ptr` in the `WorkOrder` automatically destroys the content.